### ISSUE TITLE
elemental2.dom.Window is missing document variable

### ISSUE THREAD

**Issue by mseele** [google/elemental2#60]
I'm trying to access the document from a elemental2.dom.HTMLIFrameElement but noticed that elemental2.dom.Window (via HTMLIFrameElement.contentWindow) has no document variable. 
There is no chance to access an iframe document via elemental2.

This should be possible. Please see more details in the example at https://developer.mozilla.org/en-US/docs/Web/API/HTMLIFrameElement/contentWindow

I need to solve that via native gwt code:

```
public final native elemental2.dom.Document document(elemental2.dom.Window contentWindow) /*-{
	return contentWindow.document;
}-*/;
```

**Comment by jDramaix** [comment]
**Contribution is welcome here.**

This can be easily fixed by 
 - adding the `document` property to the `Window` prototype in [google/closure-compiler/blob/master/externs/browser/html5.js](https://github.com/google/closure-compiler/blob/master/externs/browser/html5.js)

**Comment by softweaver** [comment]
We already have:

import static elemental2.dom.DomGlobal.document;
import static elemental2.dom.DomGlobal.window;

**Comment by jDramaix** [comment]
You are right but DomGlobal.document and DomGlobal.window are targeting the global document and window objects. The goal is to access document from any Window object that could potentially comes from an IFrame.

**Comment by lofidewanto** [comment]
I have a question to this problem. So, at the moment I cannot register an event to an iFrame, is that correct?

Thanks

**Comment by jDramaix** [comment]
If you need to access the body, you can still work around this problem by using jsinterop-base:
```java
import jsinterop.base.Js;

HTMLBodyElement iframeBody = Js.asPropertyMap(iframe.contentWindow).get("body");
iframeBody.addEventListener(...)
```
